Mathlib/RingTheory/AlgebraicIndependent/Adjoin.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2021 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import Mathlib.FieldTheory.Adjoin
+import Mathlib.RingTheory.AlgebraicIndependent.Defs
+
+/-!
+# Algebraic Independence
+
+This file concerns adjoining an algebraic independent family to a field.
+
+## Main definitions
+
+* `AlgebraicIndependent.aevalEquivField` - The canonical isomorphism from the rational function
+  field ring to the intermediate field generated by an algebraic independent family.
+
+* `AlgebraicIndependent.reprField` - The canonical map from the intermediate field generated by an
+  algebraic independent family into the rational function field. It is the inverse of
+  `AlgebraicIndependent.aevalEquivField`.
+-/
+
+noncomputable section
+
+open Function Set Subalgebra MvPolynomial Algebra
+
+open scoped Classical
+
+universe u v w
+
+variable {ι : Type*} {ι' : Type*} (R : Type*) {K : Type*}
+variable {A : Type*} {A' : Type*}
+variable (x : ι → A)
+variable [CommRing R] [CommRing A] [CommRing A'] [Algebra R A] [Algebra R A']
+
+variable {R} {x}
+
+open AlgebraicIndependent
+
+namespace AlgebraicIndependent
+
+section reprField
```

Comment:
Do we still need this section? It has the same scope as the `namespace` section enclosing it.